Getting error while installing  ```mdbook-svgbob```  dependency

![image](https://github.com/google/comprehensive-rust/assets/65899331/85191700-01f5-4d18-83c0-17c3c590790c)

Getting error while installing  ```mdbook-svgbob```  dependency .All other dependencies  got installed.

I tried in ```windows-11``` and  ```Ubuntu (WSL)```

Full error is:
```
error[E0308]: mismatched types
   --> /usr/local/google/home/djmitche/.cargo/registry/src/index.crates.io-6f17d22bba15001f/mdbook-svgbob-0.2.1/src/preprocessor.rs:105:21
    |                   
104 |                         match (&e, &mut state) {                
    |                               ---------------- this expression has type `(&pulldown_cmark::Event<'_>, &mut process_code_blocks::State)`
105 |                             (Start(CodeBlock(Fenced(Borrowed(mark)))), None) if mark == &cfg.code_block => {
    |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `pulldown_cmark::Event<'_>`, found a different `pulldown_cmark::Event<'_>`
    |                   
    = note: `pulldown_cmark::Event<'_>` and `pulldown_cmark::Event<'_>` have similar names, but are actually distinct types
note: `pulldown_cmark::Event<'_>` is defined in crate `pulldown_cmark`                                
   --> /usr/local/google/home/djmitche/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pulldown-cmark-0.9.6/src/lib.rs:229:1         
    |
229 | pub enum Event<'a> {               
    | ^^^^^^^^^^^^^^^^^^                
note: `pulldown_cmark::Event<'_>` is defined in crate `pulldown_cmark`
   --> /usr/local/google/home/djmitche/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pulldown-cmark-0.10.0/src/lib.rs:353:1
    |                           
353 | pub enum Event<'a> {                                                                                                                                                                                                                          
    | ^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `pulldown_cmark` are being used?
```
This is fixed in GitHub actions by using `--locked`:
```yaml
    steps:                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
      # The --locked flag is important for reproducible builds. It also                                                                                                                                                                                                                                                                                                                                                                                                                                  
      # avoids breakage due to skews between mdbook and mdbook-svgbob.                                                                                                                                                                                                                                                                                                                                                                                                                                   
      - name: Install mdbook                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
        run: cargo install mdbook --locked --version 0.4.36                                                                                                                                                                                                                                                                                                                                                                                                                                              
        shell: bash                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
      - name: Install mdbook-svgbob                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
        run: cargo install mdbook-svgbob --locked --version 0.2.1                                                                                                                                                                                                                                                                                                                                                                                                                                        
        shell: bash                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
```
so that should work here. I'll add an update to `CONTRIBUTING.md` and `README.md` as necessary.

--- a/README.md
+++ b/README.md
@@ -65,10 +65,10 @@ Then install these tools with:
 
 ```shell
 cargo install mdbook
-cargo install mdbook-svgbob
-cargo install mdbook-i18n-helpers
-cargo install --path mdbook-exerciser
-cargo install --path mdbook-course
+cargo install --locked mdbook-svgbob
+cargo install --locked mdbook-i18n-helpers
+cargo install --locked --path mdbook-exerciser
+cargo install --locked --path mdbook-course
 ```
 
 Run

